projects/storefrontapp/src/environments/environment.ts

```diff
@@ -6,4 +6,5 @@
 export const environment = {
   production: false,
   occBaseUrl: '',
+  occApiPrefix: '',
```

Comment:
we have to set a default value here because it won't work without unless we can add a default value in the proxyconfig properties for it as well like how it is for occBaseUrl

Reply:
Done

Reply:
Done